plugins/Satellites/src/gui/SatellitesDialog.hpp

```diff
@@ -138,6 +138,8 @@ private slots:
 	void setRightSideToROMode();
 	void setRightSideToRWMode();
 
+	void enableMinMaxAltitude(bool state);
+
 private:
 	//! @todo find out if this is really necessary... --BM
```

Comment:
Any resolution here? (Not strictly part of the PR, but this question is over 9 years old now...)

Reply:
I didn’t checked Bogdan’s comment, maybe situation is gone already